openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -18,6 +18,67 @@ package com.iabtechlab.openrtb.v2;
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+message Transparency {
+  // Domain of the entity that applied user parameters.
+  optional string domain = 1;
+
+  // Array for platform or sell-side use of any user parameters (using the list
+  // provided by DSA Transparency Taskforce). Note; See definition and list of
+  // possible user parameters as listed here, applied consistently in both bid
+  // request and/or bid response.
+  repeated int32 params = 2;
+
+  // Convenience values used in the 'params' field.
+  enum Params {
+    Params_UNKNOWN = 0;
+
+    // Information about the user, collected and used across contexts, that is
+    // about the user's activity, interests, demographic information, or other
+    // characteristics.
+    PROFILING = 1;
+
+    // Use of real-time information about the context in which the ad will be
+    // shown, to show the ad, including information about the content and the
+    // device, such as: device type and capabilities, user agent, URL, IP
+    // address, non-precise geolocation data. Additionally, use of basic
+    // cross-context information not based on user behavior or user
+    // characteristics, for uses such as frequency capping, sequencing, brand
+    // safety, anti-fraud.
+    BASIC_ADVERTISING = 2;
+
+    // The precise real-time geolocation of the user, i.e. GPS coordinates
+    // within 500 meter radius precision.
+    PRECISE_GEOLOCATION = 3;
+  }
+}
+
+message DsaRequest {
+  // Flag to indicate if DSA information should be made available. This will
+  // signal if the bid request belongs to an Online Platform/VLOP, such that a
+  // buyer should respond with DSA Transparency information based on the
+  // pubrender value. 0 = Not required, 1 = Supported, bid responses with or
+  // without DSA object will be accepted, 2 = Required, bid responses without
+  // DSA object will not be accepted, 3 = Required, bid responses without DSA
+  // object will not be accepted, Publisher is an Online Platform.
+  optional int32 required = 1;
```

Comment:
I see that it's been renamed to `dsarequired` in the doc https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/dsa_transparency.md#object-dsa